src/components/WorkspaceDetails/WorkspaceDetails.test.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import type { MemoryHistory } from 'history';
+import { k8sGetResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { WorkspaceDetails } from './index';
+
+expect.extend(toHaveNoViolations);
+
+jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
+  ...jest.requireActual('@openshift/dynamic-plugin-sdk-utils'),
+  k8sGetResource: jest.fn(),
+}));
+const k8sGetResourceMock = k8sGetResource as jest.Mock;
+
+const workspaceData = {
+  apiVersion: 'v1beta1',
+  apiGroup: 'tenancy.kcp.dev',
+  kind: 'Workspace',
+  metadata: {
+    name: 'demo-workspace',
+    labels: {
+      label1: 'value1',
+      label2: 'value2',
+    },
+    creationTimestamp: '2022-09-15T21:07:32Z',
+  },
+  spec: { type: { name: 'universal' } },
+};
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
+  useParams: () => ({
+    workspaceName: workspaceData.metadata.name,
+  }),
+}));
+
+describe('Workspace Details Page', () => {
+  let history: MemoryHistory;
+
+  beforeEach(() => {
+    jest.resetModules();
+    k8sGetResourceMock.mockClear();
+    history = createMemoryHistory();
+    history.push('/workspaces/demo-workspace');
+  });
+
+  it('Call to get information is made', async () => {
```

Comment:
Nit - we should investigate adding a lint rule to make sure we stay consistent with our spec function calls (use either `it` or `test` for consistency).

Reply:
Test wins.  I'll make it consistent in Hac Infra.

If you are curious here are the numbers:

HAC-Infra:  3 files use it(), 4 files use test()
SDK: 6 files use it(), 10 files use test()
HAC-Core: 0 files use it(), 2 files use test()